enhancements/sha-bundle-digest.md

```diff
@@ -0,0 +1,80 @@
+
+---
+title: SHA-Bundle-Digest
+authors:
+- "@theishshah"
+reviewers:
+- @camilamacedo86 
+- @joelanford 
+- @jmrodri
+- @ryantking 
+- @tlwu2013 
+approvers:
+- @tlwu2013 
+- @jmrodri
+- @joelanford 
+creation-date: 2022-01-14
+last-updated: 2022-01-14
+status:implementable
+---
+
+# Title
+Add support for SHA-based Digest for Images
+
+## Summary
+
+Currently, operators generated by `operator-sdk` will store and retrieve container images based on the image name and the image tag in the bundle. This EP adds support to `operator-sdk` such that images can be stored and retrieved in the bundle with an image name and an image digest as an optional replacement for a tag. Image digests are SHA-256 sums that are hashed from the image layer therefore uniquely identifying one specific image.
+
+## Motivation
+
+Users in air-gapped (disconnected) environments are currently unable to get images based on tags, and can only access local registries based on the digest.
```

Comment:
I think we should start by defining what air-gapped or disconnected means then using one term the rest of the document.

E.g.
```
Many users run Kubernetes in an air-gapped environment (also known as an offline or disconnected environment), which is when a cluster network does not have any access to the public internet, usually for security purposes. In this setup, the docker daemon is unable to fetch images based on tag because ...
```

After reading this section I should know:
a) What is an air-gapped environment?
b) Why can I not use traditional image tags in an air-gapped environment?
c) How does adding support for digest-based images fix the issue in b?

Currently, this section explains the "what," but it also needs to explain the "why."

Reply:
+1 to all of the above